wfl/Models/FoodOrderStatusDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Symphono.Wfl.Models
+{
+    public class FoodOrderStatusDto
+    {
+        public string Status { get; set; }
```

Comment:
I think the benefit of specific error messages could be useful in the future, but I don't feel extremely strongly about that benefit. I was initially passing the enum directly but changed it to avoid the situation where a user entered a number that was invalid, such as `5` when there are only 2 or 3 statuses.